ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -111,79 +137,118 @@ class TransactionsScreen extends React.Component<Props> {
+      pms.find(pm => pm.idWallet === selectedWallet.idWallet)
+    ),
+    undefined
+  );
+
+  const { present } = useRemovePaymentMethodBottomSheet({
+    icon: amex,
+    caption: I18n.t("wallet.methods.card.name")
+  });
+
+  return (
+    <WalletLayout
+      title={I18n.t("wallet.paymentMethod")}
+      allowGoBack={true}
+      topContent={headerContent(selectedWallet, isFavorite)}
+      hideHeader={true}
+      hasDynamicSubHeader={true}
+      topContentHeight={HEADER_HEIGHT}
+      contextualHelpMarkdown={contextualHelpMarkdown}
+      faqCategories={["wallet_transaction"]}
+    >
+      {pm && (
+        <>
+          <View style={IOStyles.horizontalContentPadding}>
+            <View spacer={true} extralarge={true} />
+            <PaymentMethodCapabilities paymentMethod={pm} />
+            <View spacer={true} />
+            <ItemSeparatorComponent noPadded={true} />
+          </View>
+        </>
+      )}
+      <TransactionsList
+        title={I18n.t("wallet.transactions")}
+        amount={I18n.t("wallet.amount")}
+        transactions={transactions}
+        areMoreTransactionsAvailable={props.areMoreTransactionsAvailable}
+        onLoadMoreTransactions={handleLoadMoreTransactions}
+        navigateToTransactionDetails={
+          props.navigateToTransactionDetailsScreen
+        }
+        readTransactions={props.readTransactions}
+        ListEmptyComponent={ListEmptyComponent}
+      />
+      <View style={{marginHorizontal: variables.contentPadding, marginTop: 40 }}>
+      <UnsubscribeButton
+        onPress={() => present(() => props.deleteWallet(selectedWallet.idWallet))}
+      />
+    </View>
+    <EdgeBorderComponent />
+    </WalletLayout>
+  );
+};
 
 const mapStateToProps = (state: GlobalState) => ({
+  transactions: getTransactions(state),
+  potPayments: paymentsHistorySelector(state),
+  transactionsLoadedLength: getTransactionsLoadedLength(state),
   favoriteWallet: getFavoriteWalletId(state),
+  readTransactions: state.entities.transactionsRead,
+  areMoreTransactionsAvailable: areMoreTransactionsAvailable(state),
   paymentMethods: paymentMethodsSelector(state)
 });
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
+  loadTransactions: (start: number) =>
+    dispatch(fetchTransactionsRequest({ start })),
+  navigateToTransactionDetailsScreen: (transaction: Transaction) => {
+    dispatch(readTransaction(transaction));
+    dispatch(
+      navigateToTransactionDetailsScreen({
+        transaction,
+        isPaymentCompletedTransaction: false
+      })
+    );
+  },
   setFavoriteWallet: (walletId?: number) =>
     dispatch(setFavouriteWalletRequest(walletId)),
   deleteWallet: (walletId: number) =>
     dispatch(
       deleteWalletRequest({
         walletId,
-        onSuccess: action => {
+        onSuccess: _ => {
           showToast(I18n.t("wallet.delete.successful"), "success");
-          if (action.payload.length > 0) {
-            dispatch(navigateToWalletList());
```

Comment:
Why you removed this behaviour? Is expected by the design team?

Reply:
action.payload's always not empty